middleware/kubernetes/README.md

```diff
@@ -115,6 +115,12 @@ kubernetes coredns.local {
 	# a path to a file structured like resolv.conf.
 	upstream 12.34.56.78:53
 	
+	# federation <federation-name> <federation-domain>
```

Comment:
This whole readme needs proper docs or more expansion (this is not relevant to this PR, just saying)